static/templates/settings/attachments_settings.hbs

```diff
@@ -11,7 +11,7 @@
             <thead class="table-sticky-headers">
                 <th data-sort="alphabetic" data-sort-prop="name" class="upload-file-name">{{t "File" }}</th>
                 <th class="active" data-sort="numeric" data-sort-prop="create_time">{{t "Date uploaded" }}</th>
-                <th data-sort="mentioned_in">{{t "Mentioned in" }}</th>
+                <th class="Mentioned-in" data-sort="mentioned_in">{{t "Mentioned in" }}</th>
```

Comment:
This class name doesn't match our style; let's call this `upload-mentioned-in`.

Reply:
Hey @timabbott , I have made the changes suggested by you, could you please review the PR, Thanks.